docs/01-app/03-api-reference/06-cli/create-next-app.mdx

```diff
@@ -17,33 +17,33 @@ npx create-next-app@latest [project-name] [options]
-| `--empty`                               | Initialize an empty project                                     |
-| `--use-npm`                             | Explicitly tell the CLI to bootstrap the application using npm  |
-| `--use-pnpm`                            | Explicitly tell the CLI to bootstrap the application using pnpm |
-| `--use-yarn`                            | Explicitly tell the CLI to bootstrap the application using Yarn |
-| `--use-bun`                             | Explicitly tell the CLI to bootstrap the application using Bun  |
-| `-e` or `--example [name] [github-url]` | An example to bootstrap the app with                            |
-| `--example-path <path-to-example>`      | Specify the path to the example separately                      |
-| `--reset-preferences`                   | Explicitly tell the CLI to reset any stored preferences         |
-| `--skip-install`                        | Explicitly tell the CLI to skip installing packages             |
-| `--disable-git`                         | Explicitly tell the CLI to disable git initialization           |
-| `--yes`                                 | Use previous preferences or defaults for all options            |
+| Options                                 | Description                                                           |
+| --------------------------------------- | --------------------------------------------------------------------- |
+| `-h` or `--help`                        | Show all available options                                            |
+| `-v` or `--version`                     | Output the version number                                             |
+| `--no-*`                                | Negate default options. E.g. `--no-ts`                                |
+| `--ts` or `--typescript`                | Initialize as a TypeScript project (default)                          |
+| `--js` or `--javascript`                | Initialize as a JavaScript project                                    |
+| `--tailwind`                            | Initialize with Tailwind CSS config (default)                         |
+| `--eslint`                              | Initialize with ESLint config                                         |
+| `--biome`                               | Initialize with Biome config                                          |
+| `--no-linter`                           | Skip linter configuration                                             |
+| `--app`                                 | Initialize as an App Router project                                   |
+| `--api`                                 | Initialize a project with only route handlers                         |
+| `--src-dir`                             | Initialize inside a `src/` directory                                  |
+| `--turbopack`                           | Force enable Turbopack in generated package.json (enabled by default) |
```

Comment:
Hmm... Do we need a `--webpack` flag for CNA?

Reply:
https://app.graphite.dev/submit/vercel/next.js/84476 will do this